src/neoxp/neoxp.csproj

```diff
@@ -12,6 +12,10 @@
     <PackageReference Include="Nito.Disposables" Version="2.3.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Update="Neo.BlockchainToolkit.Library" Version="3.6.19" />
+  </ItemGroup>
+
```

Comment:
`3.6.19` doesn't exists anymore its `3.6.0` now. But we have updated repo with lots of changed please merge before adding this. If still needed then keep it.